release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -438,6 +438,24 @@ Starting in {product-title} 4.14, Extended Update Support (EUS) is extended to t
 [id="ocp-4-17-networking_{context}"]
 === Networking
 
+[id="ocp-ptp-fast-events-rest-api-v2-available_{context}"]
+==== New PTP fast events REST API v2 available
+
+A new PTP fast events O-RAN v3 compliant REST API v2 is available.
```

Comment:
Is it O-RAN v3 or ORAN-3 Release 3?
I have no idea what O-RAN is, but "ORAN-3 Release 3" and "ORAN-3 Release 0030" are what I see on o-ran.org. :shrug:

Reply:
Their docs are not great. Let's go with O-RAN Release 3.